### PR TITLE
Fix typo in build-master.yaml

### PR DESCRIPTION
**Pull request by davidcassany**
Signed-off-by: David Cassany <dcassany@suse.com>

### CODE REVIEW

**Comment by davidcassany** [comment]
This is a fix for [this issue](https://github.com/rancher-sandbox/cOS-toolkit/actions/runs/1030842842)

File: .github/workflows/build-master.yaml.orig [hunk 1]
```diff
@@ -0,0 +1,1112 @@
+
```
Comment (mudler) [inline .github/workflows/build-master.yaml.orig:1]: Looks like a leftover?

File: .github/workflows/build-master_BACKUP_27191.yaml [hunk 2]
```diff
@@ -0,0 +1,1111 @@
+
```
Comment (mudler) [inline .github/workflows/build-master_BACKUP_27191.yaml:1]: ditto

File: .github/workflows/build-master.yaml.orig (hunk 1 above)
Comment (davidcassany) [reply .github/workflows/build-master.yaml.orig]: Yes I had some garbage in my checkout and I did a wild `git add`

**Review by mudler — APPROVED** [review]

**Pull request merged**